src/client.js

```diff
@@ -129,97 +125,69 @@ Client.prototype.connect = function(args) {
 	var config = Helper.getConfig();
 	var client = this;
 
+	var nick = args.nick || "lounge-user";
+
+	var network = new Network({
+		name: args.name || "",
+		host: args.host || "",
+		port: parseInt(args.port, 10) || (args.tls ? 6697 : 6667),
+		tls: !!args.tls,
+		password: args.password,
+		username: args.username || nick.replace(/[^a-zA-Z0-9]/g, ""),
```

Comment:
Yup, that's why I said "I know it was already there before but I never paid attention to that `.replace()` statement." :-)

Still, I am wondering if you knew why it's there, and if so we can add a comment above.